app/models/user.rb

```diff
@@ -12,7 +12,7 @@ class User < ApplicationRecord
 
   include PgSearch::Model
   pg_search_scope :search_user,
-    against: [ :first_name, :middle_name, :last_name, :languages ],
+    against: [ :first_name, :middle_name, :last_name, :username, :location, :company, :job_title, :languages, :skills ],
```

Comment:
1) Open for discussion but should we put all params in one single array? or should we separate the search by each param? For example, pg_search_scope :search_by_first_name, etc; Let's say the search is based on both :first_name and :location attributes for example, the single array might not suffice 

2) Suggest we add is_available (from Users) domain & years of experiences (from Expertises, so should be using associated_against)

Reply:
1. Separating the search by each parameter seems like the best option. However, in my opinion, it would be beneficial to place parameters such as location, skills, and languages within a filter option rather than the search itself.
What do you think?

2. Good idea, i will add the expertises domain to the search function.

Reply:
Thanks @Roveri91, I am fine with your proposed way